### PR TITLE
Data:Greenchapeljohn

### PR DESCRIPTION
**Pull request by greenchapeljohn**
## Changes proposed
Adding JohnGreenDev Linkfree Data file

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3566"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please keep your profile changes only and revert the other changes.
I think this can be fixed by updating your branch. Any help, let us know. 🙂

**Comment by greenchapeljohn** [comment]
My commit only contains the one file change. 
The other change has been done by the github action ( I assume setup in the project ?)

**Comment by greenchapeljohn** [comment]
The above has been succeeded  with main

**Comment by greenchapeljohn** [comment]
Think the above has now been succeeded  with main

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please rename your file to `username.json`, else some features may not work for you. 

It should be `greenchapeljohn.json`. 🛠

**Comment by greenchapeljohn** [comment]
File Is now renamed

File: data/greenchapeljohn.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    {
+      "name": "Mastodon: Connect with me",
+      "url": "https://fosstodon.org/@JohnGreenDev",
+      "icon": "FaMastodon"
+    }
+  ],
+  "milestones": [
+   
+  ]
```
Comment (Panquesito7) [inline data/greenchapeljohn.json:51]: I think it's best to remove this field if you're not going to use it, IMO.
```suggestion
  ]
```

**Comment by greenchapeljohn** [comment]
Sure, I updated that, did plan to add them later but I can stick the object back in once I have some milestones written to go in it.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by eddiejaoude** [comment]
> My commit only contains the one file change.
> The other change has been done by the github action ( I assume setup in the project ?)

@greenchapeljohn thanks for fixing 👍 it looks like the GitHub Actions are running on your fork, these should not be running on the fork (they should be off by default on a fork)

**Comment by greenchapeljohn** [comment]
Will take a look and see why they are switched on. I did have an old fork from a while back when I did some doc additions, probably had it enabled from then.

**Comment by eddiejaoude** [comment]
@greenchapeljohn ah maybe, let us know how it goes or if we can help with anything 👍

**Pull request merged**